examples/redux-observable-shopping-cart/js/services/index.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Mocking client-server processing
+ */
+import _products from './products'
+
+const TIMEOUT = 100;
+const MAX_CHECKOUT = 2; // max different items
+
+export const api = {
+  getProducts() {
+    return new Promise( resolve => {
+      setTimeout(() => resolve(_products), TIMEOUT)
+    })
+  },
+
```

Comment:
Hmm yeah I still think it should be an Observable since we're not just changing middleware--we use rxjs operators and rxjs is not redux-observable. It's best to provide examples as idiomatic rxjs as possible, even if that means it differs from redux-saga in implementation.